api/api.go

```diff
@@ -99,13 +99,13 @@ func (a *Api) routeBase(w http.ResponseWriter, req *http.Request) {
 func dumpRequest(logger *zap.Logger, boltReq *boltrouter.BoltRequest) {
 	boltDump, err := httputil.DumpRequest(boltReq.Bolt, true)
 	if err != nil {
-		logger.Error("bolt dump request", zap.Error(err))
+		logger.Error("error dumping bolt request", zap.Error(err))
```

Comment:
go errors should not have the work `error` it's redundant.

The info of the level is already in the log info, no need to add error

Reply:
done.